C# extension doesn't work under devcontainer

<!-- To prefill this information:
1. Open Visual Studio Code
2. Bring up the command palette (press <kbd>F1</kbd>)
3. Type `CSharp: Report an issue`

If the `CSharp: Report an issue` command doesn't appear, make sure that you have C# extension version 1.17.0 or newer installed.
-->

## Environment data
<details>
  <summary>`dotnet --info` output:</summary>

```  
b09f71e2c0a6:/_workspace# dotnet --info
.NET SDK (reflecting any global.json):
 Version:   5.0.400
 Commit:    d61950f9bf

Runtime Environment:
 OS Name:     alpine
 OS Version:  3.13
 OS Platform: Linux
 RID:         linux-musl-x64
 Base Path:   /usr/share/dotnet/sdk/5.0.400/

Host (useful for support):
  Version: 5.0.9
  Commit:  208e377a53

.NET SDKs installed:
  5.0.400 [/usr/share/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 5.0.9 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 5.0.9 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download  

```
</details>

VS Code version: 1.59.1
C# Extension version: v1.23.15

## OmniSharp log
<details>
  <summary>`"omnisharp.loggingLevel": "debug",`</summary>
  

```
Starting OmniSharp server at 9/3/2021, 11:20:48 AM
    Target: /_workspace/_dumps.sln

OmniSharp server started.
    Path: /root/.vscode-server/extensions/ms-dotnettools.csharp-1.23.15/.omnisharp/1.37.15/run
    PID: 13284

[ERROR] Error: OmniSharp server load timed out. Use the 'omnisharp.projectLoadTimeout' setting to override the default delay (one minute).
```

</details>

## Steps to reproduce

1. `dotnet new console`
2. Create dev container (I tried 5.0.400-alpine3.13 and 5.0.400-buster-slim dotnet sdk images)
3. Open dev container environment
4. Install C# extension there

## Expected  behavior

Omnisharp is started under a dev container

## Actual behavior

Omnisharp doesn't start.

### Additional error output

![image](https://user-images.githubusercontent.com/40987940/131998307-a02cf2de-1ff7-4756-b872-691a8aa11f03.png)

```bash
ps -a
PID   USER     TIME  COMMAND
    1 root      0:00 /bin/sh -c echo Container started trap "exit 0" 15  while sleep 1 & wait $!; do :; done
   92 root      0:00 /root/.vscode-server/bin/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/node /tmp/vscode-remote-containers-server-21929b26aee4891be33964a51c53d19882a636df.js
  109 root      0:00 sh /root/.vscode-server/bin/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/server.sh --log debug --force-disable-user-env --use-host-proxy --disable-telemetry --port 0 --extensions-download-dir /root/.vscode-server/extensionsCache --start-server --disable-websocket-compression
  111 root      0:06 /root/.vscode-server/bin/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/node /root/.vscode-server/bin/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/out/vs/server/main.js --log debug --force-disable-user-env --use-host-proxy --disable-telemetry --port 0 --extensions-download-dir /root/.vscode-server/e
  143 root      0:02 /vscode/vscode-server/bin/alpine/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/node /vscode/vscode-server/bin/alpine/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/out/bootstrap-fork --type=ptyHost
 9401 root      0:00 /root/.vscode-server/bin/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/node /tmp/vscode-remote-containers-server-1bd12ce1f85f8aeebaf5402d837160af3889198b.js
 9646 root      0:00 /bin/sh
12857 root      0:00 /bin/sh
12878 root      0:00 /bin/sh
13100 root      0:00 /root/.vscode-server/bin/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/node /tmp/vscode-remote-containers-server-9d32d59ba578629e8bb99396c91218d9a591fb35.js
13142 root      0:00 /root/.vscode-server/bin/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/node -e      const net = require('net');     process.stdin.pause();     const client = net.createConnection({ port: 41955 }, () => {      client.pipe(process.stdout);      process.stdin.pipe(client);     });     client.on('c
13162 root      0:00 /vscode/vscode-server/bin/alpine/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/node /vscode/vscode-server/bin/alpine/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/out/bootstrap-fork --type=watcherService
13173 root      0:00 /root/.vscode-server/bin/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/node -e      const net = require('net');     process.stdin.pause();     const client = net.createConnection({ port: 41955 }, () => {      client.pipe(process.stdout);      process.stdin.pipe(client);     });     client.on('c
13190 root      0:01 /vscode/vscode-server/bin/alpine/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/node /vscode/vscode-server/bin/alpine/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/out/bootstrap-fork --type=extensionHost --uriTransformerPath=/vscode/vscode-server/bin/alpine/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/ou
13305 root      0:00 /vscode/vscode-server/bin/alpine/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/node /vscode/vscode-server/bin/alpine/3866c3553be8b268c8a7f8c0482c0c0177aa8bfa/extensions/json-language-features/server/dist/node/jsonServerMain --node-ipc --clientProcessId=13190
13346 root      0:00 /bin/ash
13356 root      0:00 sleep 1
13357 root      0:00 ps -a
```

--- a/package.json
+++ b/package.json
@@ -379,7 +379,7 @@
         "x86_64"
       ],
       "installTestPath": "./.omnisharp/1.39.1-net6.0/OmniSharp.dll",
-      "platformId": "alpine-x64",
+      "platformId": "linux-musl-x64",
       "isFramework": false,
       "integrity": "13A18F07ED10B96A77EF97A51FCF2ECA740A427A1C2867C41846CE0B00AD1004"
     },
@@ -395,7 +395,7 @@
         "arm64"
       ],
       "installTestPath": "./.omnisharp/1.39.1-net6.0/OmniSharp.dll",
-      "platformId": "alpine-arm64",
+      "platformId": "linux-musl-arm64",
       "isFramework": false,
       "integrity": "C5155A59D194536684B19D65E996AB645C02BD8EB66C9C810589C55E9C263E75"
     },

--- a/src/common.ts
+++ b/src/common.ts
@@ -23,10 +23,9 @@ export function getExtensionPath() {
     return extensionPath;
 }
 
-export function getUnixTempDirectory(){
+export function getUnixTempDirectory() {
     let envTmp = process.env.TMPDIR;
-    if(!envTmp)
-    {
+    if (!envTmp) {
         return "/tmp/";
     }
 
@@ -61,9 +60,9 @@ export async function execChildProcess(command: string, workingDirectory: string
     return new Promise<string>((resolve, reject) => {
         cp.exec(command, { cwd: workingDirectory, maxBuffer: 500 * 1024 }, (error, stdout, stderr) => {
             if (error) {
-                reject(`${error}
+                reject(new Error(`${error}
 ${stdout}
-${stderr}`);
+${stderr}`));
             }
             else if (stderr && !stderr.includes("screen size is bogus")) {
                 reject(new Error(stderr));
@@ -203,7 +202,7 @@ export function isSubfolderOf(subfolder: string, folder: string): boolean {
 /**
  * Find PowerShell executable from PATH (for Windows only).
  */
- export function findPowerShell(): string | undefined {
+export function findPowerShell(): string | undefined {
     const dirs: string[] = (process.env.PATH || '').replace(/"+/g, '').split(';').filter(x => x);
     const names: string[] = ['pwsh.exe', 'powershell.exe'];
     for (const name of names) {

